.travis.yml

```diff
@@ -23,7 +23,7 @@ addons:
       - g++-7
 
 dist:
-  - trusty
+  - xenial
```

Comment:
Will test it, looks like `trusty` -> `xenial` "just work" 👍

Reply:
Applied globally: https://github.com/ingenue/hunter/commit/1fbc0423cacd3a2d6a60b9052183982c8847dc70

Reply:
Warning: it does not "just work". 
For example, I had to add `libgl1-mesa-dev` which was not by default in xenial.
See
https://github.com/pthom/hunter/commit/ca96d92dc2ff5cdf2e3f8632d17297b222589689

Reply:
> For example, I had to add libgl1-mesa-dev which was not by default in xenial

That's package specific I guess.